framework/source/class/qx/ui/form/RadioButtonGroup.js

```diff
@@ -59,6 +60,7 @@ qx.Class.define("qx.ui.form.RadioButtonGroup",
 
     // attach the listener
     this.__radioGroup.addListener("changeSelection", function(e) {
+      this.fireDataEvent("changeValue", e.getData(), e.getOldData());
       this.fireDataEvent("changeSelection", e.getData(), e.getOldData());
```

Comment:
I think one of the incompatibilities come from here. I'm deriving from `qx.ui.form.RadioButtonGroup` and add and implement the `qx.ui.form.IStringForm` interface thus adding a `value` property of type string which fires its own change event.

The idea behind my implementation was to have radio buttons which each have their own string "value". Let's say two buttons labeled "male" and "female". Depending on the users choice the string value of the radio group is "female" or "male" or null if nothing was selected.

As I read the code now this means that the value of the `qx.ui.form.RadioButtonGroup` is an instance of `qx.ui.form.RadioButton` which for sure collides with my current implementation.